### DISCUSSION TITLE
Here Share Your Projects Using the SenseCAP Indicator!

### DISCUSSION THREAD

**Discussion by Love4yzp** [Seeed-Solution/SenseCAP_Indicator_ESP32#33] [Announcements]
Hello Seeed Studio Community!

We're excited to see the innovative ways you've been using the [SenseCAP Indicator](https://www.seeedstudio.com/SenseCAP-Indicator-D1-p-5643.html). This discussion is a place for you to showcase your projects, exchange ideas, and inspire each other.

Whether you're working on a small personal project or a larger-scale endeavor, we'd love to hear about it. Share your experiences, challenges, and successes. If you have any questions or need support, feel free to ask here as well.

Let's collaborate and grow together as a community. Can't wait to see your creative applications with the SenseCAP Indicator ESP32!

Happy building!

**Comment by Love4yzp** [comment]
🌟 Check out "SenseCapConfig": A handy tool for configuring SenseCAP devices using a JSON file. It simplifies connecting to Home Assistant and managing sensors/switches. [Explore here](https://github.com/YooSHome/SenseCapConfig).

**Comment by paviro** [comment]
I just started building a firmware using ESPHome for it! https://github.com/paviro/HomeDicator

**Comment by Bigbassmann** [comment]
I've just begun mine.   [](https://github.com/Bigbassmann/ESPHome-Git)
